src/impl/conv_im2col.jl

```diff
@@ -1,14 +1,14 @@
 ## This file contains im2col-backed implementations of convolution for 2d and 3d
 ## convolutions.  Expect to see a lot of indexing.
 
-# Helper functions for flipkernel-induced dyslexia
-@inline function kernel_index(w, h, d, cdims::ConvDims{N, S, P, D, false}) where {N, S, P, D}
+# Helper function for flipkernel-induced dyslexia
+function kernel_index(w, h, d, cdims::ConvDims)
+    if flipkernel(cdims)
+        return (w, h, d)
+    end
```

Comment:
```suggestion
    flipkernel(cdims) && return (w, h, d)
```
Might as well save a couple of lines :)